teuthology/task/ansible.py

```diff
@@ -36,6 +37,60 @@ def flush(self):
         pass
 
 
+class FailureAnalyzer:
+    def analyze(self, failure_log):
+        failure_obj = yaml.safe_load(failure_log)
+        lines = set()
```

Comment:
Yeah, for de-duping. The idea is to reduce down to output that's as terse as possible because nobody reads the longer failure reasons. I think order can't matter in this case

Reply:
fair enough.